OracleJava/README.md

```diff
@@ -30,31 +30,31 @@ $ bash build.sh
 ```
 ### Parent image OS version
 
-The Oracle Java images for JDK 15 and earlier use `oraclelinux:7-slim` as the default parent image.
+The Oracle Java image for JDK 16 uses `oraclelinux:8-slim` as the parent image.
 
-It is possible to use `oraclelinux:8-slim` as the parent image by using  `Dockerfile.8-slim` rather than `Dockerfile` with docker build.
+The build script `build.sh` will tag the JDK 16 image as `16` and also as `16-oraclelinux8`.
 
-e.g. to build JDK 15 with Oracle Linux 8 rather than the default Oracle Linux 7 run
+JDK 15, JDK 11, and ServerJRE 8 use `oraclelinux:7-slim` as the default parent image but allow for optionally building on `oraclelinux:8-slim` by using `Dockerfile.8-slim` rather than `Dockerfile`.
+
+e.g. to build JDK 11 with Oracle Linux 8 rather than the default Oracle Linux 7 run
 
 ```
-$ cd ../OracleJava/15
-$ docker build --file Dockerfile.8-slim --tag oracle/jdk:15-oraclelinux8 .
+$ cd ../OracleJava/11
+$ docker build --file Dockerfile.8-slim --tag oracle/jdk:11-oraclelinux8 .
 ```
+On releases prior to JDK 16 `build.sh` can be used to build on Oracle Linux 7 or on Oracle Linux 8, by passing `8-slim`.
 
-The build script on `build.sh` can be used to build with either Oracle Linux 7 or Oracle Linux 8. To build on Oracle Linux 8 pass `8-slim` to the script: 
+e.g. 
 
 ```
+$ cd ../OracleJava/11
 $ bash build.sh 8-slim
 ```
 
-In the build script, and on the example above, the image on Oracle Linux 8 has been tagged with `15-oraclelinux8`. 
-
-The Dockerfile for creating JDK 16 images, planned to be released in March 2021, will use Oracle Linux 8, rather than Oracle Linux 7 by default.
-
 ## License
 To download and run the Oracle JDK or Server JRE, regardless of inside or outside a Docker container, you must download the binary from the Oracle website and accept the license indicated on that page.
```

Comment:
Remove the word "Docker" from this sentence

Reply:
Done